android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListViewHolder.java

```diff
@@ -113,6 +116,20 @@ public void bind(final ListItemProxy proxy, final boolean selected)
 		final int minHeight = TiConvert.toTiDimension(rawMinHeight, TiDimension.TYPE_HEIGHT).getAsPixels(itemView);
 		this.content.setMinimumHeight(minHeight);
 
+		// Handle selection checkmark.
+		if (listViewProperties.optBoolean(TiC.PROPERTY_SHOW_SELECTION_CHECK, false)
+			&& listViewProperties.optBoolean(TiC.PROPERTY_EDITING, false)
+			&& listViewProperties.optBoolean(TiC.PROPERTY_ALLOWS_SELECTION_DURING_EDITING, false)
+			&& listViewProperties.optBoolean(TiC.PROPERTY_ALLOWS_MULTIPLE_SELECTION_DURING_EDITING, false)) {
+
+			if (selected) {
+				this.leftImage.setImageDrawable(checkcircleDrawable);
+			} else {
+				this.leftImage.setImageDrawable(circleDrawable);
+			}
+			this.leftImage.setVisibility(View.VISIBLE);
```

Comment:
If the section has no items, then this code will show a checkbox in an empty row under the section header.
You can test this with your test code by changing `createSection(10, 'A')` to `createSection(0, 'A')`.

So, maybe we should only set up the `leftImage` if `section.getItemCount() > 0`?